test/FunctionalTestUtils/project.json

```diff
@@ -1,18 +1,19 @@
 {
+  "version": "1.0.1",
```

Comment:
My question is mainly about the main project.json version. I don't see any explicit versioning there so version comes from somewhere magically. Should we rev it to 1.0.1?